src/components/signup/SignUp.js

```diff
@@ -0,0 +1,162 @@
+import "./SignUp.css";
+import { FaTwitter } from "react-icons/fa";
+import { useState } from "react";
+import { axiosInstanceReg } from "../api/axios";
+import Login from "../login/Login";
+
+const SignUp = () => {
+  const [user, setUser] = useState({
+    first_name: "",
+    last_name: "",
+    username: "",
+    email: "",
+    password: "",
+    password_confirmation: "",
+  });
+  const [errMsg, setErrMsg] = useState("");
+  const [success, setSuccess] = useState(false);
+
+  const handleChange = (e) => {
+    setUser((state) => ({
+      ...state,
+      [e.target.name]: e.target.value.trim(),
+    }));
+  };
+
+  const postRequest = async () => {
+    try {
+      await axiosInstanceReg.post("/register", user);
+      setSuccess(true);
+    } catch (err) {
+      console.log(err);
+      setErrMsg(err.message);
+      setSuccess(false);
+    }
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault()
+    if (user.password !== user.password_confirmation) {
+      setErrMsg("Password does not match");
+      setSuccess(false)
```

Comment:
Why?

Reply:
I actually dont need this, let me allow backend handle it and I print the response